content/docs/docs-nav.yaml

```diff
@@ -80,25 +80,26 @@ nav:
       - API docs:
           - Custom renderers: '/docs/details/custom-api-docs-renderers/'
           - Using grpc-docs: '/docs/details/grpc-docs/'
-      - TechDocs:
-          - Using TechDocs in Roadie: '/docs/details/techdocs/'
-          - Build TechDocs Via CI: '/docs/details/techdocs/build-via-ci/'
-          - Troubleshooting TechDocs: '/docs/details/techdocs/troubleshooting/'
       - Configure GitHub Enterprise: '/docs/details/github-enterprise/'
       - GitHub app permissions: '/docs/details/github-app-permissions/'
+      - Entity Push API: '/docs/details/entity-push-api/'
```

Comment:
I'm guessing these were moved to be alphabetical - Entity Push should go above GitHub app in that case.

Also ideally this kind of clear-up task would be a separate mini-PR

Reply:
I was already in there adding my entry and it was really bothering me. Understand though, will try to control myself and do it in a separate ticket, haha.